comment/api/views.py

```diff
@@ -64,6 +65,7 @@ def post(self, request, *args, **kwargs):
         if not reaction_type:
             data = {'error': 'This is an invalid reaction type'}
             return Response(data, status=status.HTTP_400_BAD_REQUEST)
+        reaction = SetReaction.get_reaction_object(comment)
```

Comment:
Why was this line added? I don't see any further use of `reaction` object

Reply:
This is for providing reaction objects for previous comments. Earlier I had used `getorcreate` function but you transformed this in the latest release. To provide backward compatibility, this is necessary.

Reply:
I undertood what you intend to do. The code here is a little bit confusing can you please adjust it as follows:
```suggestion
        reaction_obj = SetReaction.get_reaction_object(comment)
        ReactionInstance.objects.set_reaction(
            user=request.user,
            reaction=reaction_obj,
            reaction_type=reaction_type
        )
```